lib/fortnox/api/types/model.rb

```diff
@@ -4,7 +4,7 @@ module Fortnox
   module API
     module Types
       class Model < Dry::Struct
-        constructor_type(:schema)
+        transform_types(&:omittable)
```

Comment:
Does this mean that all attributes now are `omittable`?

Reply:
Correct, but we also have forced validator within the gem that covers up for this case